en/components/drop_down_virtual.md

```diff
@@ -0,0 +1,160 @@
+```html
+<!-- drop-down-virtual.component.html -->
+<button igxButton [igxToggleAction]="dropdown" [igxDropDownItemNavigation]="dropdown">Item Series</button>
+<igx-drop-down #dropdown>
+    <div class="drop-down-virtual-wrapper" style="height: {{ itemsMaxHeight }}px;">
+        <igx-drop-down-item
+            *igxFor="let item of items; index as index; scrollOrientation: 'vertical'; containerSize: itemsMaxHeight; itemSize: itemHeight;"
+            [value]="item" [isHeader]="item.header" role="option" [disabled]="item.disabled"
+            [index]="index">
+            {{ item.name }}
+        </igx-drop-down-item>
+    </div>
+</igx-drop-down>
+<div class="selection">Selected Model: <span class="selection__name">{{ dropdown.selectedItem?.value.name }}</span></div>
+```
+The additional parameters passed to the `*igxFor` directive are:
+ - `index` - captures the index of the current item in the data set
+ - `scrollOrientation` - should always be `'vertical'`
+ - `containerSize` - the size of the virtualized container (in `px`). This needs to be enforced on the wrapping `<div>` as well
+ - `itemSize` - the size of the items that will be displayed (in `px`)
+
+In order to assure uniqueness of the items, pass `item` inside of the [`value`]({environment:angularApiUrl}/classes/igxdropdownitemcomponent.html#value) input and `index` inside of the [`index`]({environment:angularApiUrl}/classes/igxdropdownitemcomponent.html#index) input of the `igx-drop-down-item`.
+To preserve selection while scrolling, the drop down item needs to have a reference to the data items it is bound to.
+> [!NOTE]
+> For the drop down to work with a virtualized list of items, [`value`]({environment:angularApiUrl}/classes/igxdropdownitemcomponent.html#value) and [`index`]({environment:angularApiUrl}/classes/igxdropdownitemcomponent.html#index) inputs **must** be passed to all items.
+> [!NOTE]
+> It is strongly advised for each item to have an unique value passed to the `[value]` input. Otherwise, it might lead to unexpected results (incorrect selection). 
+> [!NOTE]
+> When the drop down uses virtualized items, the type of [`dropdown.selectedItem`]({environment:angularApiUrl}/classes/igxdropdowncomponent.html#selecteditem) becomes `{ value: any, index: number }`, where `value` is a reference to the data item passed inside of the `[value]` input and `index` is the item's index in the data set
+
+
+### Component Definition
+
+Inside of our component's `constructor`, we'll declare a moderately large list of items (container both headers and disabled items) which we'll be displaying in our drop-down. We also need to declare `itemHeight` and `itemsMaxHeight` (used in the template) inside of our `drop-down-virtual.component.ts` file:
```

Comment:
i guess containing instead of container ?

Reply:
Thanks for pointing it out! Addressed in the latest commit